### ISSUE TITLE
Sierra 10.12.2 can't open

### ISSUE THREAD

**Issue by jung-wan-kim** [serge-rider/dbeaver#1100]
After macOS Sierra  10.12.2 update.
 - The application "DBeaver" can't be opened.

**Comment by serge-rider** [comment]
Did yo check #95? Or this is some other issue?

**Comment by jung-wan-kim** [comment]
Yes, All most same 
I fixed problem 
Mac terminal "xattr -rc /Applications/DBeaver.app", It's work! 
Thanks~

**Comment by kj1010** [comment]
Hi - I am running into similar issues described above and in #95. I am using Mac OS 10.11.6 and attempting to install v 3.8.2. Keep receiving the message: "An error has occurred. See the log file..." I have already tried all the suggested solutions described in #95 with no luck. Any suggestions? Thanks in advance

**Comment by serge-rider** [comment]
Maybe your problem is different. Most common problem on MacOS is wrong Java version. You need to install JDK 1.7+ (not JRE!).

**Comment by kj1010** [comment]
OK, great - thanks! The Java update worked (I already had JDK, just needed to update to the latest version). Thanks again.

**Comment by lacanoid** [comment]
I ran into this very situation today (for the second time) when installing DBeaver on a fresh Mac. After some googling I finally ended back here.

The need to install JSK vs. JRE on a Mac is totally non obvious. I installed JRE 1.8 from oracle (from link on the webpage) and then JDK 1.6 from apple. before finally finding the right package (with google, after reading this thread).

Perhaps the download page could mention this under Mac downloads and provide a link to http://www.oracle.com/technetwork/java/javase/downloads/index.html where one can actually
download the appropriate package.

Better yet, have DBeaver detect this problem at startup and provide a link to open the mentioned web page, instead of a rather obscure error message.
